todoman/ui.py

```diff
@@ -224,7 +219,14 @@ def compact_multiple(self, todos):
             percent = todo.percent_complete or ''
             if percent:
                 percent = " ({}%)".format(percent)
-            urgent = " " if todo.priority in [None, 0] else "!"
+            if todo.priority == 5:
+                priority = "!!"
+            elif todo.priority == 4:
```

Comment:
Here we have to check for the entire possible range (1-4), because the task could be created by another app (Android's OpenTask, Nextcloud tasks, iCloud, Microsoft Office365, ...)